dll/win32/mshtml/script.c

```diff
@@ -839,10 +839,10 @@ static HRESULT ScriptBSC_read_data(BSCallback *bsc, IStream *stream)
 
     do {
         if(This->bsc.readed >= This->size) {
-	  void *new_buf;
-	  new_buf = heap_realloc(This->buf, This->size << 1);
-	  if(!new_buf)
-	    return E_OUTOFMEMORY;
+            void *new_buf;
+            new_buf = heap_realloc(This->buf, This->size << 1);
+            if(!new_buf)
+                return E_OUTOFMEMORY;
```

Comment:
Are there real changes here? Or is it just coding style changes?
This is hard to review.
If that's just coding style, I suggest to revert.
Furthermore, that's Wine code and will break further syncing